astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/rich-text.directive.ts

```diff
@@ -0,0 +1,76 @@
+import {
```

Comment:
Please format spacing to follow the project style: 2 spaces.

Reply:
done